### PR TITLE
pkp/pkp-lib#9126: backport for multilingual crossref metadata

### PR DESCRIPTION
**Pull request by ctgraham**
Resolves pkp/pkp-lib#9126 for 3.3.x; backport of https://github.com/pkp/crossref-ojs/pull/38

### CODE REVIEW

**Comment by ctgraham** [comment]
Failed test does not appear to relate to this change:

>1) Recommend-only editors can not publish, unpublish or create versions

https://app.travis-ci.com/github/pkp/ojs/jobs/605600542#L951-L1061

Succeeded under PHP 8.0, 7.4, etc. configs, failed under PHP 8.1 config.

**Comment by ctgraham** [comment]
Again, failed tests seem to related to a timeout and broken pipe, and not to any relevant code change; at least for these commits.

File: plugins/importexport/crossref/filter/ArticleCrossrefXmlFilter.inc.php [hunk 1]
```diff
@@ -162,10 +179,26 @@ function createJournalArticleNode($doc, $submission) {
 		}
 
 		// abstract
-		if ($abstract = $publication->getData('abstract', $locale)) {
-			$abstractNode = $doc->createElementNS($deployment->getJATSNamespace(), 'jats:abstract');
-			$abstractNode->appendChild($node = $doc->createElementNS($deployment->getJATSNamespace(), 'jats:p', htmlspecialchars(html_entity_decode(strip_tags($abstract), ENT_COMPAT, 'UTF-8'), ENT_COMPAT, 'UTF-8')));
-			$journalArticleNode->appendChild($abstractNode);
+		$abstractLanguages = array_keys($publication->getData('abstract'));
```
Comment (bozana) [inline plugins/importexport/crossref/filter/ArticleCrossrefXmlFilter.inc.php:182]: @ctgraham, this part is slightly different from the changes in main and 3_4_0 -- It considers article locale at the first place and the maximum of 10 entries. Shell the other branches consider this too?

Comment (ctgraham) [reply plugins/importexport/crossref/filter/ArticleCrossrefXmlFilter.inc.php:182]: I believe 3.3.0 still uses Crossref schema 4.3.6, which limits the number of abstracts to 10.  Crossref schema 5.3.1 has no limit on the number of abstracts, so this logic is not needed on 3.4 and later.

**Pull request merged**